qbft/spectest/tests/controller/futuremsg/test.go

```diff
@@ -66,3 +66,7 @@ func (test *ControllerSyncSpecTest) Run(t *testing.T) {
 		require.NoError(t, lastErr)
 	}
 }
+
+func (test *ControllerSyncSpecTest) GetPostState() (interface{}, error) {
+	return nil, nil
```

Comment:
even though there is a post state here, we can generate it once we start working on the respective tests